source/include/evpn_fabric_overview.rst

```diff
@@ -0,0 +1,344 @@
+EVPN VXLAN overview
+===================
+
+Ethernet Virtual Private Network (EVPN) is a standard technology that
+can be used for scaling layer 2 networks that is becoming increasingly
+popular. It combines a layer 3 (Equal Cost Multipath)
+ECMP "underlay" fabric with VXLAN "overlays" that stretch VLANs between
+switches.
+
+EVPN is typically used with a spine/leaf network architecture:
+
+.. figure:: _static/spine-leaf.png
+   :alt: Spine/leaf network architecture
+   :class: no-scaled-link
+
+This type of multipath network is resilient to failures of
+individual links or devices. A standard layer 2 (Ethernet) network
+cannot achieve this without introducing forwarding loops or disabling
+links and/or switches (e.g. Spanning Tree Protocol (STP)).
+
+Often the leaf switches will be paired, with MLAG or similar technology
+used to connect servers to both switches in a pair.
+
+Border Gateway Protocol (BGP) is used in the control plane of both the
+underlay and the overlay to exchange connectivity between the switches.
+BGP is a proven, widely used protocol that underpins exchange of routing
+information on the Internet. The underlay uses standard BGP, while the
+overlay uses Multi Protocol BGP (MP-BGP) to exchange MAC addresses and
+other information between Virtual Tunnel Endpoints (VTEPs).
+
+This network architecture is undoubtedly more complex than the standard
+layer 2 networks we have generally used in the past. It's easiest
+to build up the picture in layers.
+
+Underlay IP links
+-----------------
+
+Each leaf switch has a layer 3 (IP) /31 point to point connection to
+each spine switch. Typically we would divide up a supernet (e.g. /24)
+into multiple /31 subnets. Doing this puts the spine-leaf links into
+layer 3 mode.
+
+Example leaf interface config on Dell OS10:
```

Comment:
Should we have some variables for Dell/Cumulus/etc? I'm happy to follow up with the Cumulus part.

Reply:
Sure, let's do it in a follow up.